docs/migration.rst

```diff
@@ -33,13 +33,13 @@ can change their coordinates in-place. Illustrative code::
     >>> print(line)
     LINESTRING (0 0, 2 2)
 
-    >>> line.coords = [(0, 0), (10, 0), (10, 10)]
-    >>> print(line)
+    >>> line.coords = [(0, 0), (10, 0), (10, 10)]  # doctest: +SKIP
+    >>> print(line)  # doctest: +SKIP
```

Comment:
Would there be a way to skip this entire file, instead of having to add those many individual skips? (in the end, this whole file is a "historical document", which we don't necessarily need to keep working?)

Reply:
I investigated some options, and there isn't an easy way to exclude a file (e.g. [this one](https://stackoverflow.com/a/55044034/)). I found it simpler and easier to skip each of the breaking lines. Note that this markup does not render in the docs ([here](https://shapely--2277.org.readthedocs.build/en/2277/migration.html)).